pyvista/core/pointset.py

```diff
@@ -406,28 +412,116 @@ def is_all_triangles(self):
         --------
         >>> import pyvista
         >>> plane = pyvista.Plane()
-        >>> plane.is_all_triangles()
+        >>> plane.is_all_triangles
         False
 
         >>> sphere = pyvista.Sphere()
-        >>> sphere.is_all_triangles()
+        >>> sphere.is_all_triangles
         True
 
         """
+        class CallableBool(int):  # pragma: no cover
+            """Boolean that can be called.
+
+            Programmer note: We must subclass int and not bool
+            https://stackoverflow.com/questions/2172189/why-i-cant-extend-bool-in-python
+
+            Implemented for backwards compatibility as
+            ``is_all_triangles`` was changed to be a property in
+            ``0.32.0``.
+            """
+
+            def __new__(cls, value):
+                """Use new instead of __init__.
+
+                See:
+                https://jfine-python-classes.readthedocs.io/en/latest/subclass-int.html#emulating-bool-using-new
+
+                """
+                return int.__new__(cls, bool(value))        
+
+            def __call__(self):
+                """Return bool of self and permit calling this bool."""
```

Comment:
This docstring seems a bit confused for me. The "permit calling this bool" is semantically outside `__call__` in my head. It's the CallableBool that permits calling itself. As far as the `__call__` is concerned, it just returns bool of self.

Reply:
Quite right.  Docs unclear, fixing...